content/post/116-protoschool-merkle-dags.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2021-01-10
+url: /2021-01-14-protoschool-tutorial-merkle-dags/
+title: Explore Merkle DAGs at ProtoSchool
+author: Teri Chadbourne
+tags: ipfs, ipld, protoschool
+header_image: 116-protoschool-merkle-dags.png
+---
+
+The [ProtoSchool](https://proto.school) team is pleased to announce the launch of a shiny new multiple-choice tutorial on [Merkle DAGs](https://proto.school/merkle-dags), the cornerstone of our freshly launched [IPLD course](https;//proto.school/course/ipld). 🎉
```

Comment:
I'm still a big fan of spelling out acronyms on first reference, so in this case I'd like to see IPLD spelled out in this intro paragraph.

Reply:
Good call. Done!